storage/innobase/row/row0import.cc

```diff
@@ -2560,6 +2583,14 @@ row_import_cfg_read_index_fields(
 		field->fixed_len = mach_read_from_4(ptr) & ((1U << 10) - 1);
 		ptr += sizeof(ib_uint32_t);
 
+		if (version == IB_EXPORT_CFG_VERSION_V2) {
+			field->descending = (mach_read_from_4(ptr) == 1);
+			if (fread(row, 1, 4, file) != 4) {
+				goto file_read_error;
+			}
+			ptr = row;
+		}
```

Comment:
Can we avoid allocating 4 more bytes for this 1 flag? In any case, the `==` should be replaced with `&` so that we could freely repurpose the remaining flags later.